query-engine/connectors/sql-query-connector/src/database/operations/read.rs

```diff
@@ -148,8 +150,14 @@ where
             let _ = row.values.pop();
 
             let mut record = Record::from(row);
+
+            for (_, value) in parent_ids.iter() {
+                record.values.push(value.clone()); // we need the id there as well for some reason :shrug:
```

Comment:
We needed the parent id once in the record values and once in the `parten_id`. We can remove this code and see which part is crashing due to this.